aws/v12.1.0/announcement.md

```diff
@@ -0,0 +1,12 @@
+# :zap: Giant Swarm Release 12.1.0 for AWS :zap:
```

Comment:
@glitchcrab pls give feedback for slack announcement. Usually, it's first 2-3 paragraphs of full release notes, then links to full release notes.